[2024-08-17T03:03:25.046+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-17T03:03:25.056+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xcom_dag.branch scheduled__2024-08-16T00:00:00+00:00 [queued]>
[2024-08-17T03:03:25.061+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xcom_dag.branch scheduled__2024-08-16T00:00:00+00:00 [queued]>
[2024-08-17T03:03:25.061+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-17T03:03:25.070+0000] {taskinstance.py:2879} INFO - Executing <Task(BranchPythonOperator): branch> on 2024-08-16 00:00:00+00:00
[2024-08-17T03:03:25.073+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=367) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-17T03:03:25.074+0000] {standard_task_runner.py:72} INFO - Started process 375 to run task
[2024-08-17T03:03:25.075+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xcom_dag', 'branch', 'scheduled__2024-08-16T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpm36el0vc']
[2024-08-17T03:03:25.075+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask branch
[2024-08-17T03:03:25.104+0000] {task_command.py:467} INFO - Running <TaskInstance: xcom_dag.branch scheduled__2024-08-16T00:00:00+00:00 [running]> on host 0bd5af1ac055
[2024-08-17T03:03:25.146+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xcom_dag' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-16T00:00:00+00:00'
[2024-08-17T03:03:25.147+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-17T03:03:25.156+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-08-17T03:03:25.163+0000] {python.py:240} INFO - Done. Returned value was: t3
[2024-08-17T03:03:25.163+0000] {branch.py:38} INFO - Branch into t3
[2024-08-17T03:03:25.164+0000] {skipmixin.py:230} INFO - Following branch ('t3',)
[2024-08-17T03:03:25.172+0000] {skipmixin.py:278} INFO - Skipping tasks [('t2', -1)]
[2024-08-17T03:03:25.188+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-17T03:03:25.188+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=branch, run_id=scheduled__2024-08-16T00:00:00+00:00, execution_date=20240816T000000, start_date=20240817T030325, end_date=20240817T030325
[2024-08-17T03:03:25.215+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-17T03:03:25.235+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-17T03:03:25.237+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
